manifests/implementation/bitnami/redis/install.yaml

```diff
@@ -0,0 +1,502 @@
+ocfVersion: 0.0.1
+revision: 0.1.0
+kind: Implementation
+metadata:
+  prefix: "cap.implementation.bitnami.redis"
+  name: install
+  displayName: "install Action"
+  description: "install Action"
```

Comment:
for `manifests/implementation/aws/redis/install.yaml` it's changed, shouldn't we change it here too?